app/helpers/data_getter.py

```diff
@@ -98,7 +98,7 @@ def get_invite_by_user_id(user_id):
     @staticmethod
     def get_all_events():
         """Method return all events"""
-        return Event.query.order_by(desc(Event.id)).filter_by(in_trash=False).all()
+        return Event.query.order_by(Event.created_at).filter_by(in_trash=False).all()
```

Comment:
Shouldn't this be in `desc(created_at)` ?? Please check your code after making changes before pushing it. Thanks...